website/static/js/accountSettings.js

```diff
@@ -201,22 +201,32 @@ var UserProfileViewModel = oop.extend(ChangeMessageMixin, {
 
             this.client.update(this.profile()).done(function (profile) {
                 this.profile(profile);
-
                 var emails = profile.emails();
+                var emailAdded = false;
                 for (var i=0; i<emails.length; i++) {
                     if (emails[i].address() === email.address()) {
+                        emailAdded = true;
                         this.emailInput('');
-                        var addrText = $osf.htmlEscape(email.address());
-                        $osf.growl('<em>' + addrText  + '</em> added to your account.','You will receive a confirmation email at <em>' + addrText  + '</em>. Please check your email and confirm.', 'success');
-                        return;
                     }
                 }
+                if (emailAdded === true) {
+                    var addrText = $osf.htmlEscape(email.address());
+                    bootbox.alert({
+                                title: 'Confirm Change Notice',
```

Comment:
This dialog isn't really a confirmation; it's a notification to the user that an email was sent. Let's change this to "Confirmation email sent" (in sentence case, as per osf-style). 

I'll make this change when I merge.